pkg/graph/resolvers/trb_request_relation_test.go

```diff
@@ -12,71 +16,114 @@ func (suite *ResolverSuite) TestSetTRBRequestRelationNewSystem() {
 	ctx := suite.testConfigs.Context
 	store := suite.testConfigs.Store
 
+	ctx = appcontext.WithLogger(ctx, suite.testConfigs.Logger)
+	ctx = dataloaders.CTXWithLoaders(ctx, dataloaders.NewDataLoaders(suite.testConfigs.Store, func(ctx context.Context, s []string) ([]*models.UserInfo, error) { return nil, nil }))
```

Comment:
I might push a change up that I was playing with locally which adds default dataloaders to the `suite.testConfigs.Context` so you don't have to create this each time

@samoddball -- you good with that?

Reply:
7f73b01b7 fixes this!

Reply:
nice!!! That is awesome, huge convenience update. Thanks Clay 😄